hera_mc/correlator.py

```diff
@@ -38,6 +39,10 @@
                 'phase_switching_off': 'phase_switch_disable',
                 'noise_diode_on': 'noise_diode_enable',
                 'noise_diode_off': 'noise_diode_disable',
+                'noise_diode_enable': 'noise_diode_enable',
+                'noise_diode_disable': 'noise_diode_disable',
```

Comment:
why are you adding these lines?

Reply:
Ah yes happy to remove, but noise_diode_enable/disable are the actual corr_cm commands, and honestly I am more familiar with corr_cm than M&C so I tried to run noise_diode_enable once and had to look up the M&C command. 

So, I thought it might be nice to have the actual corr_cm commands but it _is_ just more to keep track of now.
I'm not opposed to them being different things in the end and just deleting these lines and the others you commented on.